x/bank/internal/keeper/keeper.go

```diff
@@ -236,6 +236,15 @@ func (k BaseSendKeeper) SendCoins(ctx sdk.Context, fromAddr sdk.AccAddress, toAd
 		return err
 	}
 
+	// Create account if recipient does not exist.
+	//
+	// NOTE: This should ultimately be removed in favor a more flexible approach
```

Comment:
is there an issue for this?

Reply:
@sunnya97 said he'd create an ADR for his "its on me" message idea.